src/Compute/Compute/Manual/PSVirtualMachinePatchAssessmentResult.cs

```diff
@@ -29,7 +29,7 @@ public class PSVirtualMachinePatchAssessmentResult
         public int? CriticalAndSecurityPatchCount { get; set; }
         public int? OtherPatchCount { get; set; }
         public DateTime? StartDateTime { get; set; }
```

Comment:
you need to add a breaking change suppression to this file:
tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv

and add a description to this PR describing why you need to do this and it is not really a breaking change.